tvloader/parser2v3/parse_file_test.go

```diff
@@ -425,39 +426,26 @@ func TestParser2_3CanParseFileTags(t *testing.T) {
 			len(parser.file.FileTypes))
 	}
 
-	// File Checksums
-	codeSha1 := "85ed0817af83a24ad8da68c2b5094de69833983c"
-	sumSha1 := "SHA1: 85ed0817af83a24ad8da68c2b5094de69833983c"
-	codeSha256 := "11b6d3ee554eedf79299905a98f9b9a04e498210b59f15094c916c91d150efcd"
-	sumSha256 := "SHA256: 11b6d3ee554eedf79299905a98f9b9a04e498210b59f15094c916c91d150efcd"
-	codeMd5 := "624c1abb3664f4b35547e7c73864ad24"
-	sumMd5 := "MD5: 624c1abb3664f4b35547e7c73864ad24"
-	err = parser.parsePairFromFile2_3("FileChecksum", sumSha1)
-	if err != nil {
-		t.Errorf("expected nil error, got %v", err)
+	testChecksums := map[common.ChecksumAlgorithm]string{
+		"MD5":    "624c1abb3664f4b35547e7c73864ad24",
+		"SHA1":   "85ed0817af83a24ad8da68c2b5094de69833983c",
+		"SHA256": "11b6d3ee554eedf79299905a98f9b9a04e498210b59f15094c916c91d150efcd",
+		"SHA512": "4ced3267f5ed38df65ceebc43e97aa6c2948cc7ef3288c2e5074e7df7fab544cc93339604513ea5f65616f9ed1c48581465043c8a9b693ef20fd4fddaf25e1b9",
```

Comment:
hmm, can we add 1 of the newer hashes to the test case here?

Reply:
So SHA512 is one of the added algorithms. That was the motivation to start this PR, ingesting sha512 checksummed objects was not possible before.

Reply:
See the current implementation:

https://github.com/spdx/tools-golang/blob/main/tvloader/parser2v3/parse_package.go#L114-L116

Reply:
We need to add unit tests for the actual hashed values, I'm working on that